src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -187,6 +187,8 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
       }
 
       event.preventDefault();
+      event.stopPropagation();
```

Comment:
this prevents the `compositionstart` from being fired.
( `compositionstart`  is an event that is fired when you press a key that waits for a second key press, like accents and diacritics in some layouts)